soh/src/overlays/actors/ovl_En_Reeba/z_en_reeba.c

```diff
@@ -155,6 +155,7 @@ void EnReeba_Destroy(Actor* thisx, PlayState* play) {
         if (spawner->actor.update != NULL) {
             if (spawner->curNumSpawn > 0) {
                 spawner->curNumSpawn--;
+                spawner->timer = 1 * (CVarGetInteger("gEnhancements.LeeverSpawnRate", 1) * 10);
```

Comment:
so it looks like this value is originally set here

https://github.com/HarbourMasters/Shipwright/blob/8a14fea94c07332bcf61a135076d930073e1d2da/soh/src/overlays/actors/ovl_En_Encount1/z_en_encount1.c#L70-L71

i'm curious as to why this change is here instead of there

also, with a default of 1 it seems that would override that `30` to be a default of `10`

Reply:
Fixed timer math. The other value you linked seems to only affect their initial spawn timer, but not the timing for inbetween new spawns. The other timer in that file doesn't seem to do what I want either. So unless someone else can get different results from them, I am sticking with the location I have